designer/server/src/test/scala/pl/touk/nussknacker/ui/api/helpers/NuResourcesTest.scala

```diff
@@ -224,11 +227,10 @@ trait NuResourcesTest
       )
     )
 
-  protected def deployRoute(deploymentCommentSettings: Option[DeploymentCommentSettings] = None) =
+  protected def deployRoute() =
     new ManagementResources(
       processAuthorizer = processAuthorizer,
       processService = processService,
-      deploymentCommentSettings = deploymentCommentSettings,
       deploymentService = deploymentService,
```

Comment:
You need to build deploymentService with deploymentCommentSettings. Some of the tests in ManagementResourcesSpec rely on non-empty configuration.

Reply:
I think this has been set to be empty everywhere. I've only deleted the empty constructor and everything worked ok. I will double check.